:lady_beetle: [BUG] Crash on opening app

**Describe the bug**
Quik now crashes the instant I try to open it

**To Reproduce**
1. Update the app to 4.3.2 Non-Fdroid
2. Open the app
3. Crash

**Expected behavior**
The app not to crash

**Smartphone (please complete the following information):**
 - Device: Pixel 7a
 - OS: GrapheneOS
 - App Version: 4.3.2-release

**Additional context**
Crash log from Graphene:
```
type: crash
osVersion: google/lynx/lynx:16/BP3A.251105.015/2025120401:user/release-keys
flags: dev options enabled
package: dev.octoshrimpy.quik:2234, targetSdk 33
process: dev.octoshrimpy.quik
processUptime: 186 + 302 ms
installer: dev.imranr.obtainium

java.lang.RuntimeException: Unable to start activity ComponentInfo{dev.octoshrimpy.quik/dev.octoshrimpy.quik.feature.main.MainActivity}
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4336)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4522)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:222)
	at android.app.servertransaction.TransactionExecutor.executeNonLifecycleItem(TransactionExecutor.java:133)
	at android.app.servertransaction.TransactionExecutor.executeTransactionItems(TransactionExecutor.java:103)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:80)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2839)
	at android.os.Handler.dispatchMessage(Handler.java:110)
	at android.os.Looper.dispatchMessage(Looper.java:315)
	at android.os.Looper.loopOnce(Looper.java:251)
	at android.os.Looper.loop(Looper.java:349)
	at android.app.ActivityThread.main(ActivityThread.java:9085)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:593)
	at com.android.internal.os.ExecInit.main(ExecInit.java:50)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:371)
Caused by: io.realm.exceptions.RealmMigrationNeededException: Migration is required due to the following errors:
- Property 'Conversation.sendAsGroup' has been added.
- Property 'Message.sendAsGroup' has been added.
	at io.realm.internal.OsSharedRealm.nativeGetSharedRealm(Native Method)
	at io.realm.internal.OsSharedRealm.<init>(SourceFile:0)
	at io.realm.internal.OsSharedRealm.getInstance(SourceFile:0)
	at io.realm.BaseRealm.<init>(SourceFile:0)
	at io.realm.BaseRealm.<init>(SourceFile:0)
	at io.realm.Realm.<init>(SourceFile:0)
	at io.realm.Realm.createInstance(SourceFile:0)
	at io.realm.RealmCache.createInstance(Unknown Source:4)
	at io.realm.RealmCache.doCreateRealmOrGetFromCache(Unknown Source:91)
	at io.realm.RealmCache.createRealmOrGetFromCache(SourceFile:0)
	at io.realm.Realm.getDefaultInstance(Unknown Source:28)
	at dev.octoshrimpy.quik.repository.ConversationRepositoryImpl.getConversations(SourceFile:0)
	at dev.octoshrimpy.quik.repository.ConversationRepository$DefaultImpls.getConversations$default(Unknown Source:7)
	at dev.octoshrimpy.quik.feature.main.MainViewModel.<init>(Unknown Source:207)
	at dev.octoshrimpy.quik.feature.main.MainViewModel_Factory.provideInstance(Unknown Source:142)
	at dev.octoshrimpy.quik.feature.main.MainViewModel_Factory.get(SourceFile:0)
	at dev.octoshrimpy.quik.feature.main.MainViewModel_Factory.get(SourceFile:0)
	at dev.octoshrimpy.quik.feature.main.MainActivityModule_ProvideMainViewModelFactory.provideInstance(Unknown Source:0)
	at dev.octoshrimpy.quik.feature.main.MainActivityModule_ProvideMainViewModelFactory.get(SourceFile:0)
	at dev.octoshrimpy.quik.feature.main.MainActivityModule_ProvideMainViewModelFactory.get(SourceFile:0)
	at dev.octoshrimpy.quik.common.ViewModelFactory.create(SourceFile:0)
	at androidx.lifecycle.ViewModelProvider$Factory$-CC.$default$create(SourceFile:0)
	at dev.octoshrimpy.quik.common.ViewModelFactory.create(SourceFile:0)
	at androidx.lifecycle.ViewModelProvider.get(SourceFile:0)
	at androidx.lifecycle.ViewModelProvider.get(SourceFile:0)
	at dev.octoshrimpy.quik.feature.main.MainActivity$viewModel$2.invoke(SourceFile:0)
	at dev.octoshrimpy.quik.feature.main.MainActivity$viewModel$2.invoke(SourceFile:0)
	at kotlin.SynchronizedLazyImpl.getValue(Unknown Source:20)
	at dev.octoshrimpy.quik.feature.main.MainActivity.getViewModel(Unknown Source:2)
	at dev.octoshrimpy.quik.feature.main.MainActivity.onCreate(Unknown Source:12)
	at android.app.Activity.performCreate(Activity.java:9297)
	at android.app.Activity.performCreate(Activity.java:9275)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1534)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4320)
	... 16 more
```

--- a/data/src/main/java/com/moez/QKSMS/migration/QkRealmMigration.kt
+++ b/data/src/main/java/com/moez/QKSMS/migration/QkRealmMigration.kt
@@ -37,7 +37,7 @@ class QkRealmMigration @Inject constructor(
 ) : RealmMigration {
 
     companion object {
-        const val SCHEMA_VERSION: Long = 14
+        const val SCHEMA_VERSION: Long = 15
     }
 
     @SuppressLint("ApplySharedPref")
@@ -247,19 +247,6 @@ class QkRealmMigration @Inject constructor(
                 .addField("isRegex", Boolean::class.java, FieldAttribute.REQUIRED)
                 .addField("includeContacts", Boolean::class.java, FieldAttribute.REQUIRED)
 
-            realm.schema.get("Conversation")
-                ?.addField("sendAsGroup", Boolean::class.java, FieldAttribute.REQUIRED)
-                ?.transform { conversation ->
-                    conversation.setBoolean(
-                        "sendAsGroup",
-                        (conversation.getList("recipients").size > 1)
-                    )
-                }
-
-            realm.schema.get("Message")
-                ?.addField("sendAsGroup", Boolean::class.java, FieldAttribute.REQUIRED)
-
-
             version++
         }
 
@@ -292,6 +279,23 @@ class QkRealmMigration @Inject constructor(
             realm.createObject("EmojiSyncNeeded")
 
             version++
+        }
+
+        if (version == 14L) {
+            if (realm.schema.get("Conversation")?.hasField("sendAsGroup") == false) {
+                realm.schema.get("Conversation")
+                    ?.addField("sendAsGroup", Boolean::class.java, FieldAttribute.REQUIRED)
+                    ?.transform { conversation ->
+                        conversation.setBoolean(
+                            "sendAsGroup",
+                            (conversation.getList("recipients").size > 1)
+                        )
+                    }
+            }
+            if (realm.schema.get("Message")?.hasField("sendAsGroup") == false) {
+                realm.schema.get("Message")
+                    ?.addField("sendAsGroup", Boolean::class.java, FieldAttribute.REQUIRED)
+            }
         }
 
         check(version >= SCHEMA_VERSION) {

